students/toddschultz/session03/mailroom.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+
+global donors
+donors = ["Baby Huey", 1123.00, 456.00, 1789.00, "Mighty Mouse", 99.99, "Fred Flintstone", 5550.00, 5555.00, "Road Runner", 199999.00, "Papa Smurf", 1001.00, 1002.00, 1003.00]
+
+def prompt():
+    action = input("Would you like to (S)end a Thank You, (C)reate a Report or (Q)uit? ")
+    if action.lower() == "s":
+        thank_you()
+    elif action.lower() == "c":
+        create_report()
+    elif action.lower() == "q" or action.lower() == "quit":
+        print("Have a nice day!")
+    else:
+        print("Please enter S, C, or Q.")
+        prompt()
```

Comment:
a better alternative to calling `prompt()` all over the place is to have a while True loop so that it forever loops until program is terminated.